client/app/account/billing/confirmTerminate/billing-confirmTerminate.controller.js

```diff
@@ -1,57 +1,81 @@
-angular.module("Billing.controllers").controller("Billing.controllers.TerminateServiceCtrl", function ($stateParams, BillingTerminate) {
-    "use strict";
+angular.module("Billing.controllers")
+    .controller("Billing.controllers.TerminateServiceCtrl", function ($q, $stateParams, BillingTerminate, User) {
+        "use strict";
 
-    this.reasons = ["LACK_OF_PERFORMANCES", "TOO_EXPENSIVE", "TOO_HARD_TO_USE", "NOT_RELIABLE", "NOT_NEEDED_ANYMORE", "MIGRATED_TO_COMPETITOR", "MIGRATED_TO_ANOTHER_OVH_PRODUCT", "OTHER"];
+        this.reasons = [
+            "LACK_OF_PERFORMANCES",
+            "TOO_EXPENSIVE",
+            "TOO_HARD_TO_USE",
+            "NOT_RELIABLE",
+            "NOT_NEEDED_ANYMORE",
+            "MIGRATED_TO_COMPETITOR",
+            "MIGRATED_TO_ANOTHER_OVH_PRODUCT",
+            "OTHER"
+        ];
 
-    this.serviceId = $stateParams.id;
-    this.serviceState = null;
-    this.token = $stateParams.token;
-    this.loading = true;
-    this.terminating = false;
-    this.error = false;
-    this.globalError = null;
+        this.serviceId = $stateParams.id;
+        this.serviceState = null;
+        this.token = $stateParams.token;
+        this.loading = true;
+        this.userLoading = true;
+        this.terminating = false;
+        this.error = false;
+        this.globalError = null;
 
-    this.init = function () {
-        if (!this.token || !this.serviceId) {
-            this.globalError = true;
-            return;
-        }
+        this.init = function () {
```

Comment:
It would be nice to convert it in es6 class and use `$onInit`   ;)

Reply:
I don't think you should do this automatically as it would be sidetracking from the actual feature, BUT, as the file looks simple enough, I acknowledge that it would be nice 😄

Reply:
It would be nice. Sometime we're said to do it, sometime we're said to not. I don't know anymore what is decided on each manager.

Reply:
Depending of the context (bugfix/feature vs hotfix) but improving code to a fashion ES6 style is always appreciate (that's same with components from OVH UI Kit).

Personaly if submitting a hotfix, I prefer to stay with the minimal diff as possible.

Otherwise if you're targeting a script with 500+ lines of code you might consider to do it in a separate pull request :)